lib/domain/entities/gps_coordinates.dart

```diff
@@ -14,8 +14,11 @@ class GpsCoordinates with _$GpsCoordinates {
     /// Longitude of position between -90 (included) and 90 (included) degrees
     double longitudeInDegrees,
   ) = _GpsCoordinates;
+
   const GpsCoordinates._();
 
+  factory GpsCoordinates.paris() => const GpsCoordinates(48.864716, 2.349014);
```

Comment:
Being very chauvin here, but I like using the "point zero des routes de Frances" as a default location :)
```suggestion
  factory GpsCoordinates.paris() => const GpsCoordinates(48.85341, 2.3488);
```